lib/elixir_search_extractor_web/controllers/api/v1/keyword_controller.ex

```diff
@@ -0,0 +1,40 @@
+defmodule ElixirSearchExtractorWeb.Api.V1.KeywordController do
+  use ElixirSearchExtractorWeb, :controller
+
+  alias ElixirSearchExtractor.SearchKeyword.SearchKeywords
+  alias ElixirSearchExtractorWeb.Api.V1.ErrorView
+  alias ElixirSearchExtractorWeb.Api.V1.Keywords.{DetailView, ListView}
+
+  def index(conn, params) do
+    {keywords, pagination} =
+      SearchKeywords.paginated_user_keywords(conn.assigns.current_user, params)
+
+    conn
+    |> put_view(ListView)
+    |> render("index.json", %{data: keywords, meta: meta_data(pagination)})
+  end
+
+  def show(conn, %{"id" => keyword_id}) do
+    case SearchKeywords.get_user_keyword(conn.assigns.current_user, keyword_id) do
+      nil ->
+        conn
+        |> put_status(:unprocessable_entity)
```

Comment:
Should this return `not_found` status instead?

```suggestion
        |> put_status(:not_found)
```